src/data/monsters.txt

```diff
@@ -2543,6 +2552,7 @@ five skeleton invaders	1005	bigskeleton5.gif	Atk: 150 Def: 135 HP: 150 Init: 50
 ten skeletons	1006	skel10.gif	Atk: 300 Def: 270 HP: 300 Init: 50 P: undead	bag of bones (100)
 25 skeletons	1007	skel25.gif	NOCOPY NOMANUEL Atk: 700 Def: 630 HP: 700	2 bag of bones (m100)
 100 skeletons	1008	skel100.gif	NOCOPY NOMANUEL Atk: 1500 Def: 1350 HP: 1500	4-6 bag of bones (m100)
+Rene C. Corman (The Cannon Museum)	1009	reneccorman.gif	NOCOPY NOMANUEL Atk: 500 Def: 500 HP: 500 Manuel: "Rene C. Corman" Wiki: "Rene C. Corman (The Cannon Museum)"	Stabonic scroll (n100)
```

Comment:
the goon is distinguished by date, so probably "Rene C. Corman (2010)"

Reply:
but then again, considering the other one is evergreen, not sure.

Reply:
year doesn't disambiguate very well, but neither does the containing zone. Went with the path and the event name as the most relevant contexts.

Reply:
https://github.com/kolmafia/kolmafia/pull/2132/commits/d7e20bbd6a5996fae119a6ef38c23d8055dbeabf